### PR TITLE
Load a folder of views

### LINKED ISSUE #13: View not detected
Hi,
I am trying out your package, and I think it's an awesome idea. I was playing around with this idea of putting an entire django app into one file, like this:

```
import sys

from django.conf import settings
from django.core.wsgi import get_wsgi_application
from django.urls import path
from django_view_decorator import include_view_urls

settings.configure(
    DEBUG=True,
    ROOT_URLCONF=__name__,
    SECRET_KEY="don't look me",
    INSTALLED_APPS=["app"],
)

urlpatterns = [
    path("", include_view_urls()),
]

application = get_wsgi_application()

if __name__ == "__main__":
    from django.core.management import execute_from_command_line

    execute_from_command_line(sys.argv)
```

And in the folder `app`, I have a `views.py` file with following view:

```
@view(paths="", name="index")
def index(request):
    return HttpResponse("hello")
```

But when I start the app with `python main.py runserver`, the view is not detected and I only see the django startproject view with the rocket.
Why is that? Is there a setting I am missing?

my directory structure looks like this:
```
.
├── app
│   ├── __init__.py
│   └── views.py
└── main.py
```

This are my package versions:
```
Package               Version
--------------------- -------
asgiref               3.7.2
Django                4.2.3
django-view-decorator 0.0.4
gunicorn              21.2.0
packaging             23.1
pip                   22.2.2
setuptools            63.2.0
sqlparse              0.4.4
typing_extensions     4.7.1
```

### PR DESCRIPTION
**Pull request by Mte90**
This fix https://github.com/valberg/django-view-decorator/issues/13

In urls.py it is required that code:

```
import os
root = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
urlpatterns = [
path("", include_view_urls(extra_modules=[root + "/portal/views"])),
]
```

In this way loads from a folder the various views automatically.
I will update the readme tomorrow (also some parameters like login_required are not documented, I will do that).

### CODE REVIEW

**Comment by Mte90** [comment]
Now the readme is updated but require a review from @valberg

**Comment by valberg** [comment]
Hi @Mte90 

Thanks for the pull request.

I'm not sure that this is the right approach. I would much rather that we traverse the contents of a python module than directly interacting with the filesystem. I think that could be accomplished by using `pkg_util` from the standard library.

Also I think we should be explicit about how the functionality is used. One should be explicit about wanting to scan all modules in a package, so we should introduce a new keyword argument "extra_packages" to solve this case.

Basically I want the API to be:

```
include_view_urls(
    extra_modules=["dotted.path.to.module"],
    extra_packages=["dotted.path.to.package"],
)
```

File: README.md [hunk 1]
```diff
@@ -61,6 +61,26 @@ class FooList(ListView):
     model = Foo
 ```
 
+### Parameters
+
+* `paths`: is in a plural form because support also a list of paths
+* `name` the classic route name
+* `namespace`: {I don't know what to write here}
```
Comment (Mte90) [inline README.md:68]: it is required a review here

**Comment by Mte90** [comment]
So I added the new parameter and updated the readme but about `pkg_util` I don't know how to proceed.

To me is better if we don't know need to to define a variable with the root folder so it should be more fast butI don't know how to achieve it, I am not so much experience in this python knowledge.

**Comment by Mte90** [comment]
Ping @valberg

**Comment by Mte90** [comment]
ping @valberg

**Comment by Mte90** [comment]
ping @valberg

**Comment by Mte90** [comment]
Why you closed it?
I don't see that you implemented that feature

**Comment by Mte90** [comment]
didn't got any answer
